programs/dual-farming/src/lib.rs

```diff
@@ -844,11 +848,12 @@ pub struct ClosePool<'info> {
         constraint = pool.reward_duration_end > 0,
         constraint = pool.reward_duration_end < sysvar::clock::Clock::get().unwrap().unix_timestamp.try_into().unwrap(),
         constraint = pool.user_stake_count == 0,
+        constraint = pool.total_staked == 0
```

Comment:
Could we provide a function to withdraw all mistakenly sent token when duration ends? 
We can pay back user off-chain

Reply:
Added